python/ray/tests/test_basic.py

```diff
@@ -2627,7 +2628,7 @@ def f():
 
 
 def test_specific_job_id():
-    dummy_driver_id = ray.JobID(b"00112233445566778899")
+    dummy_driver_id = ray.JobID(b"0010")
```

Comment:
better use `from_int` here. because that's the public api that should be used. 
or we can change the constructor to accept an integer.